redaxo/src/core/lib/console/setup/check.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+/**
+ * @package redaxo\core
+ *
+ * @author staabm
+ *
+ * @internal
+ */
+class rex_command_setup_check extends rex_console_command
+{
+    protected function configure()
+    {
+        $this
+            ->setDescription('Check the commandline interface (CLI) environment for REDAXO requirements')
+        ;
+    }
+
+    protected function execute(InputInterface $input, OutputInterface $output)
+    {
+        $io = $this->getStyle($input, $output)->getErrorStyle();

```

Comment:
Ist es hier sinnvoll, STDERR zu nutzen?
Bisher machen wir es nur in einem Fall, wo die eigentliche Ausgabe PHP-Code ist.
Aber hier sind die Messages ja eher die eigentliche Ausgabe.

Reply:
copy & paste.. hab gar nicht drüber nachgedacht... hier brauchen wir kein STDERR